docs/user_guide/emummc/partitioning_sd.md

```diff
@@ -91,13 +105,12 @@
 !!! danger "SD card identification"
 	Make 100% sure you get your microSD card during the following steps. If you are not careful, you can end up wiping the wrong disk. We are not responsible for data loss when doing this.
 
-### What you will need (macOS)
-
-- Access to an administrator account.
+!!! tip "What you will need (macOS)"
+    - Access to an administrator account.
 
 ### Instructions (macOS)
 
-1. Open Terminal at `/Applications/Terminal.app`.
+1. Open Terminal by hitting `Command + Space` to bring up spotlight then type `terminal` and hit enter.
```

Comment:
This is not a good measure to find something. Specifically, if a user has any other program called `Terminal`, this search result will cause unpredictable results. `/Applications/Terminal.app` is clear and always returns predictable results.

Reply:
Reverted.